### PR TITLE
DDS automatic folder maker

### PR DESCRIPTION
**Pull request by llZektorll**
## Edited
Creation of folders according to DDS automatically 

## Checklist
- [x] before creating a new note, I searched the vault
- [x] new notes have the `.md` extension
- [x] (if applicable) attached images have descriptive file names
- [x] (if applicable) for new notes in the folder "04 - Guides, Workflows, & Courses", I added a link to them in one of the "for {group X}" overviews: https://publish.obsidian.md/hub/04+-+Guides%2C+Workflows%2C+%26+Courses/%F0%9F%97%82%EF%B8%8F+04+-+Guides%2C+Workflows%2C+%26+Courses

### CODE REVIEW

File: 02 - Community Expansions/02.05 All Community Expansions/Auxiliary Tools/DSS_Folder_Maker.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+aliases:DSS_Folder_Maker
```
Comment (claremacrae) [inline 02 - Community Expansions/02.05 All Community Expansions/Auxiliary Tools/DSS_Folder_Maker.md:2]: For my own education, what's the benefit of giving an alias that is (as far as I can see) the same as the file name?

File: 02 - Community Expansions/02.05 All Community Expansions/Auxiliary Tools/DSS_Folder_Maker.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+publish: true
+---
+
+# DSS_Folder_Maker
+
+Available for: [[Windows Tools|Windows]]
+
+Available on: 
+
+  Obsidian Forum: <a href="https://forum.obsidian.md/t/ddc-automatic-folder-creation/31216"Obsidian Forum</a>
```
Comment (claremacrae) [inline 02 - Community Expansions/02.05 All Community Expansions/Auxiliary Tools/DSS_Folder_Maker.md:15]: This link gives me an "Oops this page doesn't exist.."

`https://forum.obsidian.md/t/ddc-automatic-folder-creation/31216"Obsidian Forum`

Does the `"` make it do something different in Obsidian?

**Review by claremacrae — COMMENTED** [review]
Thank you very much... Reading the changes in GitHub, I made a couple of comments.

**Comment by llZektorll** [comment]
Due to some error, i will close the pull request and send a new one once they are all corrected

**Pull request closed without merging**